Error with MacOS

## Problem

The GitHub Action *Hiden-dependency-updater* doesn't work on MacOS.   
The log of the error:
```log
  temp_result=$(curl -s https://api.github.com/repos/MathieuSoysal/file-updater-for-release/tags | grep -h "name" | grep -h "" |  head -1 | grep -ho "v\?[0-9.]*")
  echo "new-tag=${temp_result}" >> $GITHUB_OUTPUT
  shell: /bin/bash --noprofile --norc -e -o pipefail {0}
Error: Process completed with exit code 1.
```
*Full log is [here](https://github.com/MathieuSoysal/Hiden-dependancy-updater/actions/runs/4179145016/jobs/7238879809)*


The commit that is guilty is [this](https://github.com/MathieuSoysal/Hiden-dependancy-updater/commit/7ee0edb8f2841e7943d7be5b87493a04816139f7)

## Solution

The problem is probably come from `grep -h ""`.

--- a/action.yml
+++ b/action.yml
@@ -33,7 +33,7 @@ inputs:
   selector: # selector to get a particular kind of tag in many kind of tags
     description: "Selector to get the latest tag"
     required: true
-    default: ""
+    default: ":"
     type: string
 
 

